src/xb-silo.c

```diff
@@ -435,6 +435,8 @@ xb_silo_get_node_tail(XbSilo *self, XbSiloNode *n)
 const gchar *
 xb_silo_get_node_element(XbSilo *self, XbSiloNode *n)
 {
+	if (n == NULL)
+		return NULL;
```

Comment:
I believe it'll be a good idea to be proactive and to catch all the `xb_node_get_` functions in this merge request, not only those I happened to use.